src/components/Footer/FooterLinks/index.jsx

```diff
@@ -0,0 +1,138 @@
+import elo from './icons/elo.svg';
+import mastercard from './icons/mastercard.svg';
+import paypal from './icons/paypal.svg';
+import visa from './icons/visa.svg';
+import telephone from './icons/telephone.svg';
+import email from './icons/email.svg';
+import {
+    FooterContainer,
+    FooterCategories,
+    CategoryLinks,
+    SupportInfo,
+    FooterImages,
+} from './styles';
+
+const FooterLinks = [
+    {
+        title: 'Categories',
+        links: [
+            {
+                title: 'Peças',
+                path: 'xxx',
+            },
+            {
+                title: 'Serviços',
+                path: 'xxx',
+            },
+            {
+                title: 'Manutenção',
+                path: 'xxx',
+            },
+            {
+                title: 'Sistema de Trocas',
+                path: 'xxx',
+            },
+        ],
+    },
+    {
+        title: 'Institucional',
+        links: [
+            {
+                title: 'Quem Somos',
+                path: 'xxx',
+            },
+            {
+                title: 'Nosso Historia',
+                path: 'xxx',
+            },
+            {
+                title: 'Trabalhe Conosco',
+                path: 'xxx',
+            },
+        ],
+    },
+    {
+        title: 'Ajuda',
+        links: [
+            {
+                title: 'Política de Privacidade',
+                path: 'xxx',
+            },
+            {
+                title: 'Política de Troca',
+                path: 'xxx',
+            },
+            {
+                title: 'Termos e Condições',
+                path: 'xxx',
+            },
+            {
+                title: 'FAQ',
+                path: 'xxx',
+            },
+            {
+                title: 'Fale Conosco',
+                path: 'xxx',
+            },
+        ],
+    },
+    {
+        title: 'Suporte',
+        links: [
+            {
+                number: '080000000',
+                email: 'ajuda@scania.com.br',
+                emailicon: [<FooterImages src={email} height="20" width="20" />],
+                numbericon: [<FooterImages src={telephone} height="20" width="20" />],
+                img: [
+                    <FooterImages width="30" height="30" src={boleto} />,
+                    <FooterImages width="30" height="30" src={elo} />,
+                    <FooterImages width="30" height="30" src={mastercard} />,
+                    <FooterImages width="30" height="30" src={paypal} />,
+                    <FooterImages width="30" height="30" src={visa} />,
+                ],
+            },
+        ],
+    },
+];
+
+const MainFooter = () => {
+    return (
+        <FooterContainer>
+            {FooterLinks &&
+                FooterLinks.map((FooterLinks, index) => {
+                    return (
+                        <dl key={index}>
+                            {' '}
+                            <FooterCategories key={index}>{FooterLinks.title}</FooterCategories>
```

Comment:
- Same point about plural here.
- You're using a `dl` tag as a wrapper, so this `FooterCategories` should be a `dt` instead of an `h2`
- This `key` attribute is not necessary, normally we use this only on the first child inside a loop (map), in your case the `dl` above